animations/index.md

```diff
@@ -1,7 +1,21 @@
-# Module content
+# Animations in React Native
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+## Table of contents
```

Comment:
I would add gestures to topics too

Reply:
Thank you for the suggestion. Could you please describe the mentioned gesture handling topic in more details? My current idea is to highlight gesture related animations in the appropriate topics (e.g Animated API -> Examples of animations -> animations on scroll, animations to follow touch), but I may miss something here. I'll be glad to hear your thoughts.

Reply:
ok, got it, I agree with you